internal/schema-manager/mysql/mysql.go

```diff
@@ -114,6 +116,20 @@ func getDatabaseDataForSchemaDiff(
 		return nil
 	})
 
+	trigmu := sync.Mutex{}
+	errgrp.Go(func() error {
```

Comment:
is this running in a loop or is anything else other than this goroutine writing to the triggers slice?? if not I think the mutex is not needed.